snap/snapcraft.yaml

```diff
@@ -101,7 +107,7 @@ parts:
     plugin: nil
     override-pull: |
       sudo apt --assume-yes install software-properties-common
-      sudo add-apt-repository -yn ppa:mir-team/rc
+      sudo add-apt-repository -yn ppa:mir-team/dev
```

Comment:
Shouldn't we, by default, use `release`, and have CI systems install the other PPAs, as appropriate? That's what we do today?

Reply:
Oops! Missed that.